app/components/create_invite_button/create_invite_button.css

```diff
@@ -1,3 +1,3 @@
-.CreateInviteButton-success {
+.CreateInviteButton:not(.CreateInviteButton--primary) .CreateInviteButton-success {
```

Comment:
~Why is `:not(.CreateInviteButton--primary)` necessary?~ OK, understood.

Reply:
Is this what you where referring in this comment?

> I'd prefer not to add specializations that we don't use.